isic/settings.py

```diff
@@ -31,6 +31,10 @@ class IsicMixin(ConfigMixin):
 
     @staticmethod
     def mutate_configuration(configuration: ComposedConfiguration) -> None:
+        for app in ["rest_framework", "rest_framework.authtoken", "drf_yasg"]:
+            if app in configuration.INSTALLED_APPS:
+                configuration.INSTALLED_APPS.remove(app)
```

Comment:
S3FF REST endpoints still depend on DRF. Does this break those?

Reply:
It works by happenstance in that S3FF uses only basic python functions from DRF. I'll leave DRF in INSTALLED_APPS out of an abundance of caution.